lib/src/puppetdb-cli.cc

```diff
@@ -31,49 +33,111 @@ version()
 
 string
 read_config(const string& config_path) {
+    const string defaulted_config_path = config_path.empty() ?
+            "~/.puppetlabs/client-tools/puppetdb.conf" : config_path;
     const string expanded_config_path
-    { futil::tilde_expand("~/.puppetlabs/client-tools/puppetdb.conf") };
+    { futil::tilde_expand(defaulted_config_path) };
```

Comment:
Why is this default expressed late, and not in the main options area? I can't see what the default is when I do a -h atm for example, might be nice.